sdk/dotnet/State/ArtifactoryRemoteStateReferenceArgs.cs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2016-2020, Pulumi Corporation.
+
+using Pulumi.Serialization;
+
+namespace Pulumi.Terraform.State
+{
+    /// <summary>
+    /// The configuration options for a Terraform Remote State stored in the Artifactory backend. 
+    /// </summary>
+    public class ArtifactoryRemoteStateReferenceArgs : RemoteStateReferenceArgs
```

Comment:
Not really for this PR - but looking forward slightly to having a schema-driven code generator that could replace this codegen - is this pattern of subclasses of the args bag with default values on some legs representable in our schema model today?  Really question for @pgavlin?